mifosng-android/src/main/java/com/mifos/mifosxdroid/online/groupdetails/GroupDetailsFragment.java

```diff
@@ -283,7 +283,9 @@ public boolean onOptionsItemSelected(MenuItem item) {
             case R.id.group_clients:
                 mGroupDetailsPresenter.loadGroupAssociateClients(groupId);
                 break;
-
+            case R.id.add_group_saving_account:
+                mListener.addGroupSavingAccount(groupId);
```

Comment:
Couldn't this be accomplished by using just `FragmentManager`, like Document and LoanAccount fragments are added, instead of using Listener? @therajanmaurya @ishan1604 Could you please clarify which one should be followed?